src/components/modals/Export.tsx

```diff
@@ -413,7 +413,7 @@ const ModalExport: FC<{ simplePaths: SimplePath[] }> = ({ simplePaths }) => {
         e.key === 'c' &&
         (isMac ? e.metaKey : e.ctrlKey) &&
         exportContent &&
-        // do not override copy shortcut if user has text selected
+        // do not override copy command if user has text selected
```

Comment:
This should still be "shortcut" since it is referring specifically to the keyboard shortcut.